components/src/jsMain/kotlin/dev/fritz2/components/alert.kt

```diff
@@ -0,0 +1,280 @@
+package dev.fritz2.components
+
+import dev.fritz2.binding.RootStore
+import dev.fritz2.binding.SimpleHandler
+import dev.fritz2.components.validation.ComponentValidationMessage
+import dev.fritz2.components.validation.Severity
+import dev.fritz2.dom.html.RenderContext
+import dev.fritz2.styling.StyleClass
+import dev.fritz2.styling.params.BasicParams
+import dev.fritz2.styling.params.styled
+import dev.fritz2.styling.theme.*
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+
+/**
+ * A component to display an alert consisting of an icon, title and description.
+ * Different styles based on severities are supported, as well as a number of different layout options.
+ *
+ * Currently the following severities are available:
+ * - Info
+ * - Success
+ * - Warning
+ * - Error
+ * Specifying a severity will change the alert's color scheme based on the colors defined in the application theme.
+ * If no severity is specified, 'info' will be used by default.
+ *
```

Comment:
Is the background color the only thing that changes on specifying a severity? If so, shouldn't this be a property with context and accept all ColorProperties (and a different name, backgroundColor maybe)?

Reply:
Actually @jwstegemann pointed out that the icon should change as well, so I'm not sure whether I can get rid of the context. If it only was the background color that changes (as I initially though too) I would agree.